bin/set_ngrok_url

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Get current ngrok url from ngrok api launched with command "make ngrok"
+
+if docker-compose port ngrok 4040 &> /dev/null ; then
+    declare -r JQ="docker run --rm -i fundocker/jq:1.6"
+    NGROK_URL=$(
+        curl --no-progress-meter "$(docker-compose port ngrok 4040)/api/tunnels/command_line" | \
+        ${JQ} .public_url  | \
+        sed 's/"//g'
+    )
+    read -p "Apply to AWS config? (y/n) " -n 1 -r
+    echo
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        sed -i "s+^TF_VAR_marsha_base_url=.*$+TF_VAR_marsha_base_url=$NGROK_URL+" ./src/aws/env.d/development
+        make -C src/aws/ apply
```

Comment:
I don't know if it's a good idea to run the terraform apply. This is out of the scope of this script IMO. The purpose of this script is to change the base url.

Reply:
I made this for convenience : as far as I know, there's no point to run ngrok if we don't apply it to aws.
Like this, we do everything with one command.

Reply:
Yes but you can keep the convenience by creating 2 different make task and a third one running both. You are able to run one or the other. I think in some cases I would like to just update the ngrok url, work a little bit more on terraform and then apply everything.

Reply:
Ok, I see :+1: